paperqa/prompts.py

```diff
@@ -98,9 +98,12 @@
 # Prompt templates for use with LitQA
 QA_PROMPT_TEMPLATE = "Q: {question}\n\nOptions:\n{options}"
 EVAL_PROMPT_TEMPLATE = (
-    "Extract the single letter answer from the following question and answer"
```

Comment:
I think this is a solid iteration of the eval prompt template to address #709.

But I don't think this addresses [Empty session.answer for exhaustive search without an answer #723](https://github.com/Future-House/paper-qa/issues/723)

We still need the ability to inform the user of what happened at this step -- they just get a blank answer object currently. If the "user" is the eval script (as here) then this takes care of it.

Reply:
Yea, exactly -- after the runner finishes, we need some sentinel to be added. The blank isn't intuitive enough for users to understand (especially via command line or UI). Can def be another PR.

Reply:
In cases like this eval though -- the sentinel will create a need for another prompt, so we could make the sentinel a module level variable and check for it in the eval method.